<?php
try {
    if ($divisor == 0) {
        throw new Exception("DivisiÃ³n por cero.");
    }
    $resultado = $dividendo / $divisor;
} catch (Exception $e) {
    echo "Se ha producido el siguiente error: ".$e->getMessage();
}